main.go

```diff
@@ -1,21 +1,73 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"os/user"
+	"path/filepath"
 
 	"github.com/abs-lang/abs/repl"
+	"github.com/hashicorp/go-getter"
 )
 
 var Version = "1.8.0"
 
+func check(e error) {
+	if e != nil {
+		panic(e)
+	}
+}
+
 // The ABS interpreter
 func main() {
 	args := os.Args
 	if len(args) == 2 && args[1] == "--version" {
 		fmt.Println(Version)
 		return
 	}
+
+	if args[1] == "--get" {
+		if len(args) < 3 {
```

Comment:
My suggestion would be to move all of this code to an install.go module so that we don't clutter main.go. Eventually if we use something like cobra it should also become easier to take that module and covert it to cobra